corehq/apps/app_manager/templates/app_manager/no_longer_supported.html

```diff
@@ -16,7 +17,9 @@
     <ul class="nav nav-hq-sidebar"></ul>
   </nav>
 {% endblock %}
-{% block app-content %}
-  <h1>This app is no longer supported</h1>
-  <p>If you believe this to be in error, please contact support.</p>
+{% block page_content %}
+  {% blocktrans %}
```

Comment:
nit: i generally prefer to blocktrans INSIDE of block tags like `<p>` and `<h1>` tags, as that relies on the translator to get those tags (and closing tags) correct in the translated text...

Reply:
makes sense, fixed in e9065ef1f6f